src/components/BookAppRedux.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+const Book = (props) => {
+  const dispatch = useDispatch();
+  const handleDelete = () => {
+    dispatch(removeBook(props.item_id))
+    console.log(props.item_id);
```

Comment:
- [ ] We avoid leaving `console.log` in our code as it makes our codebase unprofessional so kindly consider removing it.